examples/advanced/tidb-cluster.yaml

```diff
@@ -461,6 +461,7 @@ spec:
 
     ## if enabled, the slow log will be shown in a separate sidecar container
     # separateSlowLog: true
+    # slowLogVolumeName: ""
```

Comment:
I guess this line is missed in PR https://github.com/pingcap/tidb-operator/pull/3731

Reply:
Please also update this file for the change in this PR.

Reply:
Yep, I plan to but have no idea where should I put them.